package haptix.comm.msgs;

/// \ingroup haptix_comm_msgs
/// \interface hxTime
/// \brief A translation and orientation constructed of a hxVector3 and
/// hxQuaternion.

import "hxVector3.proto";
import "hxQuaternion.proto";

message hxTransform
{
  required hxVector3 pos       = 1;
  required hxQuaternion orient = 2;
}
